### PR TITLE
add AUTHORS and CHANGES

### LINKED ISSUE #83: add AUTHORS file
Add a file with chronological list of authors (see e.g., MDAnalysis [AUTHORS](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS)).

- [x] add file
- [x] add authors + GitHub handles retroactively
- [x] add instructions for new contributors ("add yourself to AUTHORS")

### LINKED ISSUE #84: add CHANGES file
Add a file with summary of changes for each release (see e.g., MDAnalysis [CHANGELOG](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/CHANGELOG)).

- [x] add CHANGES
- [x] add releases with major changes retroactively (see [releases](https://github.com/alchemistry/alchemlyb/releases)

### PR DESCRIPTION
**Pull request by orbeckst**
- close #83
- close #84
- files were manually created based on the history (git log --format="format:%an %ad" --date="format:%Y" 0.1.0..HEAD | sort | uniq)
  and the merged PRs and closed issues

### CODE REVIEW

**Comment by orbeckst** [comment]
Hello authors of alchemlyb, please let me know if you want any changes to the AUTHOR and CHANGES file that we are finally adding. You can add a review or a comment. If I don't hear anything I assume everything is fine.

cc @orbeckst @dotsdl  @ianmkenney @shuail @trje3733 @brycestx @harlor @vtlim @lee212

**Comment by orbeckst** [comment]
@davidlmobley @mrshirts @dotsdl I need at least one approval to merge this. Multiple oks would be even better as nailing down authorship information is always something that can get tricky later. Thanks.

**Comment by orbeckst** [comment]
ping @davidlmobley @mrshirts – if you could have a very quick look and approve?

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. -*- coding: utf-8 -*-
+
+======================
+ Authors of alchemlyb
+======================
+
+The alchemlyb project is lead by Oliver Beckstein (@orbeckst), David
```
Comment (davidlmobley) [inline AUTHORS]: "lead" -> "led". :)

**Comment by orbeckst** [comment]
ping @mrshirts – you're the last one who needs to say aye or nay.

**Pull request merged**